### ISSUE TITLE
Support for Tables

### ISSUE THREAD

**Issue by revnode** [bangle-io/bangle-io#221]
The only thing that's missing for me is markdown support for tables: https://www.markdownguide.org/extended-syntax/#tables

Would you accept a PR adding this as an extension?

**Comment by kepta** [comment]
Bangle.io does support reading of tables but does not support modifying them. 

You will need to look into adding that in https://bangle.dev/docs/api/table/ , couple of other prosemirror ecosystem library have code that we can look into for inspiration.

**Comment by hyrahul** [comment]
I would love to contribute is this issues still open ? @kepta

**Comment by Harshiths1** [comment]
How would you feel about incorporating a feature that allows users to create tables using three or more hyphens (---) to define each column's header and pipes (|) to separate columns? This enhancement would ensure that the output is presented effectively, accommodating varying cell widths.

**Comment by kepta** [comment]
@Harshiths1 thats a good idea but I haven't seen other WYSIWYG markdown editors take this approach of creating tables like that. Can you provide any example?

Regarding this topic I think we can take the conventional approach of how google docs allows you to create table:

![image](https://github.com/bangle-io/bangle-io/assets/6966254/ca57f637-a10b-41a2-8581-33f272bbc4d0)

**Comment by Harshiths1** [comment]
Indeed, the traditional approach is typically self-explanatory and intuitive , also 
What are your thoughts on introducing the capability to render mathematical expressions, akin to 'KaTeX'?

![image](https://github.com/bangle-io/bangle-io/assets/136854105/ec7ef415-14da-4581-ad82-65be0eb56f06)

**Comment by kepta** [comment]
Yes it is in the radar, someone posted an issue already https://github.com/bangle-io/bangle-io/issues/197 we can talk about supporting math over there.

**Comment by regression1607** [comment]
I would like to contribute is this issues still open ?
